bench/proxygen/optimized/HQFramer.ll

```diff
@@ -503,18 +506,17 @@ sw.bb:                                            ; preds = %_ZN5folly8OptionalI
   %hasValue.i.i.i.i.i = getelementptr inbounds nuw i8, ptr %agg.result, i64 8
   store i64 %2, ptr %agg.result, align 8
   store i8 1, ptr %hasValue.i.i.i.i.i, align 8
+  store i8 1, ptr %which_.i.i.i8, align 8
   br label %return
 
 sw.epilog:                                        ; preds = %_ZN5folly8OptionalISt4pairImmEEptEv.exit5
   store i8 0, ptr %agg.result, align 8
   %hasValue.i.i.i.i.i7 = getelementptr inbounds nuw i8, ptr %agg.result, i64 8
   store i8 0, ptr %hasValue.i.i.i.i.i7, align 8
+  store i8 1, ptr %which_.i.i.i8, align 8
   br label %return
 
 return:                                           ; preds = %sw.epilog, %sw.bb, %if.then
-  %.sink = phi i8 [ 1, %sw.epilog ], [ 1, %sw.bb ], [ 2, %if.then ]
-  %which_.i.i.i8 = getelementptr inbounds nuw i8, ptr %agg.result, i64 16
-  store i8 %.sink, ptr %which_.i.i.i8, align 8
```

Comment:
Regression: Early hoisting of `%which_.i.i.i8 = getelementptr inbounds nuw i8, ptr %agg.result, i64 16` hinders the sinking of these code?